src/utils/styleHelpers.ts

```
@@ -0,0 +1,330 @@
+
+  return {
+    ...baseStyle,
+    ...sizeStyles[size],
+  };
+};
+
+// Helper function to combine multiple styles
+export const combineStyles = (...styles: (Style | undefined | null | false)[]): Style => {
+  return styles
+    .filter((s): s is Style => !!s)
+    .reduce<Style>((acc, style) => ({ ...acc, ...style } as Style), {} as Style);
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)